src/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/Tags.hpp

```diff
@@ -29,4 +70,62 @@ struct ConstraintGamma1 : db::SimpleTag {
 struct ConstraintGamma2 : db::SimpleTag {
   using type = Scalar<DataVector>;
 };
-}  // namespace GeneralizedHarmonic::ConstraintDamping::Tags
+
+/*!
+ * \brief A DampingFunction to compute the constraint damping paramter
+ * \f$\gamma_0\f$.
+ */
+template <size_t VolumeDim, typename Fr>
+struct DampingFunctionGamma0 : db::SimpleTag {
+  using DampingFunctionType =
+      ::GeneralizedHarmonic::ConstraintDamping::DampingFunction<VolumeDim, Fr>;
+  using type = std::unique_ptr<DampingFunctionType>;
+  using option_tags =
+      tmpl::list<::GeneralizedHarmonic::ConstraintDamping::OptionTags::
+                     DampingFunctionGamma0<VolumeDim, Fr>>;
+
+  static constexpr bool pass_metavariables = false;
+  static type create_from_options(const type& damping_function) noexcept {
+    return std::move(damping_function->get_clone());
```

Comment:
Does the `std::move` actual do anything here? You're already getting an rvalue result from `get_clone`

Reply:
Nope! Thanks for pointing this out, as it directed me to clear up lvalue vs rvalue. I found this page very helpful for this: https://en.cppreference.com/w/cpp/language/value_category